### DISCUSSION TITLE
How to add selected element in a bottom instead top in Ant Design Transfer?

### DISCUSSION THREAD

**Discussion by bhalsodkhushal** [ant-design/ant-design#30817] [Q&A]
https://ant.design/components/transfer/

When we move selected element to righ side at that time it add by default top of the list.

I want to add selected element in a bottom instead of top in Ant Design Transfer component.

**Comment by huntdream** [comment]
You can change  the order of `targetKeys`

**Reply by bhalsodkhushal** [reply]
how can i do it ??
